src/contracts/contract_deployer.star

```diff
@@ -16,31 +16,25 @@ def deploy_factory_contract(
 ):
     factory_deployment_result = plan.run_sh(
         name="op-deploy-factory-contract",
-        description="Deploying L2 factory contract to L1 (needs to wait for l1 to finalize, about 4 min for minimal preset, 30 min for mainnet)",
+        description="Deploying L2 factory contract to L1 (takes about a minute)",
         image=IMAGE,
         env_vars={
-            "WEB3_PRIVATE_KEY": str(priv_key),
-            "FUND_VALUE": "10",
+            "PRIVATE_KEY": str(priv_key),
+            "FUND_VALUE": "10ether",
             "DEPLOY_CONFIG_PATH": "/workspace/optimism/packages/contracts-bedrock/deploy-config/getting-started.json",
             "DEPLOYMENT_CONTEXT": "getting-started",
         }
         | l1_config_env_vars,
         run=" && ".join(
             [
-                "web3 transfer $FUND_VALUE to {0}".format(FACTORY_DEPLOYER_ADDRESS),
-                "sleep 3",
+                "cast send {0} --value $FUND_VALUE --rpc-url $L1_RPC_URL --private-key $PRIVATE_KEY".format(FACTORY_DEPLOYER_ADDRESS),
```

Comment:
At this point it is still possible that your L1 hasn't had genesis yet. Lets wait for at least slot 1 before we try to send a cast transaction. 

```suggestion
                "while true; do sleep 1; echo 'L1 Chain is starting up'; if [ \"$(curl -s $CL_RPC_URL/eth/v1/beacon/headers/ | jq -r '.data[0].header.message.slot')\" != \"0\" ]; then echo 'L1 Chain has started!'; break; fi; done",
                "cast send {0} --value $FUND_VALUE --rpc-url $L1_RPC_URL --private-key $PRIVATE_KEY".format(FACTORY_DEPLOYER_ADDRESS),
```